src/pages/welcome/index.js

```diff
@@ -30,7 +32,10 @@ const Welcome = () => {
       profile.firstName,
       profile.lastName,
       profile.githubUsername,
-      profile.bio
+      profile.bio,
```

Comment:
What's the reasoning behind providing these args to onCreateProfile?

the method signature for this func declared in src/context/auth.js shows: 

```js
  const handleCreateProfile = async (firstName, lastName, githubUrl, bio) => {
  
  }
  ```

Reply:
Thought they maybe needed for the profile page and the fields team 4 were creating, but if that's not the case can simply remove them